components/BottomNavigation/src/private/MDCBottomNavigationItemView.m

```diff
@@ -383,6 +383,7 @@ - (void)setTitle:(NSString *)title {
 -(void)setTitleVisibility:(MDCBottomNavigationBarTitleVisibility)titleVisibility {
   _titleVisibility = titleVisibility;
   [self updateLabelVisibility];
+  [self centerLayoutAnimated:YES];
```

Comment:
Does it not work to call `setNeedsLayout` here instead so any layout related to visibility can be performed in a standard UIKit layout pass?

Reply:
Yes, that's better. I'm actually going to move that into updateLabelVisibility, since that seems like a better place for that to live.

Reply:
Sounds good! 👍